spec/cms/models/training/question_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.describe Training::Question, :cms, type: :model do
+  subject(:question) { described_class.find_by(name: '1-1-4').load.first }
+
+  let(:alpha) { Training::Module.by_name(:alpha) }
+
+  before do
+    skip 'WIP' unless Rails.application.cms?
+  end
+
+  describe 'attributes' do
+    it '#options' do
+      expect(question.options.count).to eq(2)
+      expect(question.options.first).to eq(OpenStruct.new(id: 1, label: 'Correct answer 1', correct?: true))
+    end
+
+    it '#correct_answer' do
+      expect(question.correct_answer).to eq(1)
+    end
+
+    describe 'contentful attributes' do
+      it '#page_type' do
+        expect(question.page_type).to eq 'formative_questionnaire'
+      end
+
+      it '#assessment_type' do
+        expect(question.assessments_type).to eq 'formative_assessment'
+      end
+
+      it '#submodule' do
+        expect(question.submodule).to eq 1
+      end
+
+      it '#page_number' do
+        expect(question.page_number).to eq 1
+      end
+
+      it '#total_questions' do
```

Comment:
I don't think this attribute is relevant now that `Questionnaire` is retired. I think it's really "number of answers/options" and dropping it avoids the content team needing to remember to increment if they add a new one. Is there a reason for not using the multi-dimensional array format they'd prefer?

Reply:
So there'd be a second phase to this where the JSON format changes to `[[],[]]`, we validate it when instantiating the structs, and fix up the uploader with a new migration, run that in a new empty env whilst content test out what we currently have.